frappe/public/js/frappe/form/control.js

```diff
@@ -26,16 +26,27 @@ frappe.ui.form.Control = Class.extend({
 		}
 	},
 	make: function() {
+		this.make_container();
 		this.make_wrapper();
+		this.container.attr("data-fieldtype", this.df.fieldtype);
 		this.$wrapper
 			.attr("data-fieldtype", this.df.fieldtype)
 			.attr("data-fieldname", this.df.fieldname);
 		this.wrapper = this.$wrapper.get(0);
 		this.wrapper.fieldobj = this; // reference for event handlers
 	},
 
+	make_container: function() {
```

Comment:
Adding a new class might be dangerous and have collateral damage. Just fix it for image field for now.